### PR TITLE
Update index.md

### LINKED ISSUE #21487: document that ASP.NET Core doesn’t use or read app pool configuration or system.web sections.
Per @jkotalik 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec484002-18c1-6a0d-00de-820d56f038d3
* Version Independent ID: 7903ce53-2cbb-f083-7479-c8abc30211a0
* Content: [Host ASP.NET Core on Windows with IIS](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/?view=aspnetcore-5.0)
* Content Source: [aspnetcore/host-and-deploy/iis/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/iis/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #21487

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -527,14 +527,14 @@ To set environment variables for individual apps running in isolated app pools (
...
-Configuration sections of ASP.NET 4.x apps in *web.config* aren't used by ASP.NET Core apps for configuration:
+Configuration sections of ASP.NET apps in *web.config* aren't used by ASP.NET Core apps for configuration:
 
 * `<system.web>`
 * `<appSettings>`
 * `<connectionStrings>`
 * `<location>`
 
-ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index).
+ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](dotnet/core/run-time-config/)
```
Comment (wadepickett) [inline aspnetcore/host-and-deploy/iis/index.md:537]:
```suggestion
ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](xref:dotnet/core/run-time-config/)
```

**Review by wadepickett — CHANGES_REQUESTED** [review]

File: aspnetcore/host-and-deploy/iis/index.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/iis/index.md:537]:
```suggestion
ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](/dotnet/core/run-time-config/)
```

Comment (wadepickett) [reply aspnetcore/host-and-deploy/iis/index.md]: Hmm, just occurred to me that I'm not sure how xref works across repos, so I may have that wrong, looking for an example...

File: aspnetcore/host-and-deploy/iis/index.md [hunk 2]
```diff
@@ -527,14 +527,14 @@ To set environment variables for individual apps running in isolated app pools (
 
 ## Configuration sections of web.config
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/iis/index.md:528]: ## Sections that aren't used by ASP.NET Core

File: aspnetcore/host-and-deploy/iis/index.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/iis/index.md:537]:
```suggestion
ASP.NET Core apps are configured using other configuration providers. For more information, see [Configuration](xref:fundamentals/configuration/index) and [.NET Core run-time configuration settings](/dotnet/core/run-time-config/)
```

Comment (Rick-Anderson) [reply aspnetcore/host-and-deploy/iis/index.md:537]: I just forgot the leading `/`

**Review by wadepickett — APPROVED** [review]

**Pull request merged**